packages/ui/src/helpers/extensions.ts

```diff
@@ -0,0 +1,26 @@
+export const extensions = [
```

Comment:
This file is duplicating what we are importing [here](https://github.com/Polkadex-Substrate/polkadex-ts/blob/f8e3b89c346205f867ec93b9c32937291380653a/packages/react-providers/src/ExtensionsProvider/provider.tsx#L2C15-L2C15).

Reply:
Removed